extensions/rich_text_components/Image/directives/ImageDirective.js

```diff
@@ -71,18 +71,14 @@ oppia.directive('oppiaNoninteractiveImage', [
           };
           $scope.loadImage();
         } else {
+          $scope.isLoadingIndicatorShown = false;
+          $scope.isTryAgainShown = false;
           // This is the case when user is in exploration editor. We don't have
           // loading indicator or try again button for showing images in the
           // exploration editor. So we directly fetch the images from the
           // AssetsBackendApiService's cache.
```

Comment:
Change the comment accordingly. (same for ImageClickInput.js)

Reply:
Done!